protocol/src/test/java/com/amazon/opendistroforelasticsearch/sql/protocol/response/format/JdbcResponseFormatterTest.java

```diff
@@ -0,0 +1,121 @@
+ *    express or implied. See the License for the specific language governing
+ *    permissions and limitations under the License.
+ *
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.protocol.response.format;
+
+import static com.amazon.opendistroforelasticsearch.sql.data.model.ExprValueUtils.tupleValue;
+import static com.amazon.opendistroforelasticsearch.sql.data.type.ExprCoreType.ARRAY;
+import static com.amazon.opendistroforelasticsearch.sql.data.type.ExprCoreType.INTEGER;
+import static com.amazon.opendistroforelasticsearch.sql.data.type.ExprCoreType.STRING;
+import static com.amazon.opendistroforelasticsearch.sql.data.type.ExprCoreType.STRUCT;
+import static com.amazon.opendistroforelasticsearch.sql.elasticsearch.data.type.ElasticsearchDataType.ES_TEXT;
+import static com.amazon.opendistroforelasticsearch.sql.elasticsearch.data.type.ElasticsearchDataType.ES_TEXT_KEYWORD;
+import static com.amazon.opendistroforelasticsearch.sql.executor.ExecutionEngine.Schema;
+import static com.amazon.opendistroforelasticsearch.sql.executor.ExecutionEngine.Schema.Column;
+import static com.amazon.opendistroforelasticsearch.sql.protocol.response.format.JsonResponseFormatter.Style.COMPACT;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.amazon.opendistroforelasticsearch.sql.common.antlr.SyntaxCheckException;
+import com.amazon.opendistroforelasticsearch.sql.exception.SemanticCheckException;
+import com.amazon.opendistroforelasticsearch.sql.protocol.response.QueryResult;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.gson.JsonParser;
+import org.junit.jupiter.api.Test;
+
+class JdbcResponseFormatterTest {
+
+  private final JdbcResponseFormatter formatter = new JdbcResponseFormatter(COMPACT);
+
+  @Test
+  void format_response() {
+    QueryResult response = new QueryResult(
+        new Schema(ImmutableList.of(
+            new Column("name", "name", STRING),
+            new Column("address1", "address1", ES_TEXT),
+            new Column("address2", "address2", ES_TEXT_KEYWORD),
+            new Column("location", "location", STRUCT),
+            new Column("employer", "employer", ARRAY),
+            new Column("age", "age", INTEGER))),
+        ImmutableList.of(
+            tupleValue(ImmutableMap.of("name", "John", "age", 20))));
+
+    assertJsonEquals(
+        "{"
+            + "\"schema\":["
+            + "{\"name\":\"name\",\"alias\":\"name\",\"type\":\"keyword\"},"
+            + "{\"name\":\"address1\",\"alias\":\"address1\",\"type\":\"text\"},"
+            + "{\"name\":\"address2\",\"alias\":\"address2\",\"type\":\"text\"},"
+            + "{\"name\":\"location\",\"alias\":\"location\",\"type\":\"object\"},"
+            + "{\"name\":\"employer\",\"alias\":\"employer\",\"type\":\"nested\"},"
+            + "{\"name\":\"age\",\"alias\":\"age\",\"type\":\"integer\"}"
+            + "],"
+            + "\"datarows\":[[\"John\",20]],"
```

Comment:
How to handle the missing value?

Reply:
I missed other columns here. Will add and check other formatter's UT. Thanks!

Reply:
Fixed this mismatch and added another case for missing and null value.